tests/pytorch/r2.1/hf-glue.libsonnet

```diff
@@ -0,0 +1,143 @@
+    git log -1
+    pip install datasets evaluate scikit-learn
+    sed '/torchvision/d' examples/pytorch/_tests_requirements.txt > no_vision_require.txt
+    pip install -r no_vision_require.txt
+  |||,
+  local command_copy_metrics = |||
+    gsutil -m cp -r ./tensorboard-metrics/* $(MODEL_DIR)
+  |||,
+  local hf_glue = self.hf_glue,
+  hf_glue:: common.PyTorchTest {
+    local config = self,
+    modelName: 'hf-glue',
+    paramsOverride:: {
+      scriptPath: 'examples/pytorch/xla_spawn.py',
+      tpuCores: config.accelerator.numCores,
+      trainCommand: [
+        'python3',
+        self.scriptPath,
+        '--num_cores=%d' % config.paramsOverride.tpuCores,
+        'examples/pytorch/text-classification/run_glue.py',
+        '--model_name_or_path=%s' % config.paramsOverride.model_name_or_path,
+        '--logging_dir=./tensorboard-metrics',
+        '--task_name=MNLI',
+        '--cache_dir=./cache_dir',
+        '--do_train=true',
+        '--do_eval=true',
+        '--num_train_epochs=3',
+        '--max_seq_length=128',
+        '--learning_rate=3e-5',
+        '--output_dir=MNLI',
+        '--overwrite_output_dir=true',
+        '--logging_steps=30',
+        '--save_steps=3000',
+        '--overwrite_cache=true',
+        '--debug=tpu_metrics_debug',
+        '--per_device_train_batch_size=%d ' % config.paramsOverride.per_device_train_batch_size,
+        '--per_device_eval_batch_size=%d ' % config.paramsOverride.per_device_eval_batch_size,
+      ],
+    },
+    command: utils.scriptCommand(
+      |||
+        %s
+        %s
+        %s
+      ||| % [
+        command_common,
+        utils.toCommandString(self.paramsOverride.trainCommand),
+        command_copy_metrics,
+      ]
+    ),
+    podTemplate+:: {
+      spec+: {
+        containerMap+: {
+          train+: {
+            resources+: {
+              requests: {
+                cpu: '12.0',
+                memory: '80Gi',
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+  local distilbert_base_uncased = self.distilbert_base_uncased,
+  distilbert_base_uncased:: common.Convergence {
+    modelName: 'hf-glue-distilbert-b-uc',
+    paramsOverride+:: {
+      model_name_or_path: 'distilbert-base-uncased',
+      per_device_train_batch_size: 512,
+      per_device_eval_batch_size: 512,
+    },
+    metricConfig+: {
+      sourceMap+:: {
+        tensorboard+: {
+          aggregateAssertionsMap+:: {
+            'eval/accuracy': {
+              FINAL: {
+                fixed_value: {
+                  comparison: 'GREATER',
+                  value: 0.70,
+                },
+                inclusive_bounds: false,
+                wait_for_n_data_points: 0,
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+  local tpuVm = {
+    tpuSettings+: {
+      tpuVmExports+: |||
+        export XLA_USE_BF16=$(XLA_USE_BF16)
+      |||,
+    },
+  },
+  // TODO: Remove after 2.1 release cut
```

Comment:
is there a reason why xrt is being kept in for now?

Reply:
Fixed